CMakeLists.txt

```diff
@@ -468,7 +468,7 @@ endif()
 
 set(SWIFT_BUILD_HOST_DISPATCH FALSE)
 if(SWIFT_ENABLE_DISPATCH AND NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
-  if(SWIFT_BUILD_SYNTAXPARSERLIB OR SWIFT_BUILD_SOURCEKIT)
+  if(SWIFT_INCLUDE_TOOLS AND (SWIFT_BUILD_SYNTAXPARSERLIB OR SWIFT_BUILD_SOURCEKIT))
```

Comment:
Can you add a comment here explaining why you are conditionalizing building host dispatch on SWIFT_INCLUDE_TOOLS. I think it would just be good to be explicit on the reasoning for future reference.

Reply:
Done.